src/main/tile-importer.js

```diff
@@ -74,30 +78,33 @@ TileImporter.prototype._extractTar = function (tilesPath, destPath, cb) {
   pump(read, extract, cb)
 }
 
-TileImporter.prototype.moveTiles = function (tilesPath, dir, cb) {
-  var self = this
-  if (path.extname(tilesPath) === '.tar') return this._extractTar(tilesPath, dir, cb)
-  if (path.extname(tilesPath) === '.mbtiles') {
-    var destFile = path.join(dir, path.basename(tilesPath))
-    return fs.copyFile(tilesPath, destFile, cb)
-  }
-  fs.stat(tilesPath, function (err, stat) {
+TileImporter.prototype.moveTiles = function (tilesPath, tilesDest, cb) {
+  fs.stat(tilesPath, (err, stat) => {
     if (err) return cb(err)
-    if (stat.isDirectory()) {
-      return self._moveDirectory(tilesPath, dir, cb)
-    } else return cb(new Error('Tiles format not accepted'))
+    mkdirp(tilesDest, (err) => {
+      if (err) return cb(err)
+      if (path.extname(tilesPath) === '.asar') {
+        var filename = path.basename(tilesPath)
+        return fs.copyFile(tilesPath, path.join(tilesDest, filename), cb)
+      }
+      if (stat.isDirectory()) {
+        var styleId = path.basename(tilesDest)
```

Comment:
we're still supporting directories in the code, even if we aren't exposing that through the user interface. In a future iteration we can imagine folks dragging and dropping instead of selecting from a menu, which would make this use case much easier